api/odl2.py

```diff
@@ -37,7 +37,7 @@
     from core.model.patron import Hold, Loan, Patron
 
 
-class ODL2Settings(OPDS2ImporterSettings, ODLSettings):
```

Comment:
I assume the side effect of this making username and password mandatory was on purpose

Reply:
It is intentional, I think these were changed to non-required by accident in https://github.com/ThePalaceProject/circulation/pull/1189. If you checkout https://github.com/ThePalaceProject/circulation/commit/299003ea0f3a6322dbcb6068219f4c0d27cd5612 and take a look at the config, the configuration form in this PR matches what I see there.